Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem 'resque', '~>1.21'
-gem "ffi-rzmq", "~>1.0"
+gem "ffi-rzmq", "~>2.0"
```

Comment:
I know this was already here, but shouldn't this be in the `gemspec` file? Same with `resque`?